### DISCUSSION TITLE
System chat is not visible

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#423] [Q&A] [answered]
I do /hear , /block enemy, and /buy, but i dont see any "system" or "user" reply on my in-game chat communication.
How to fix it?
But the command inserted are working perfectly btw

**Comment by winstxnhdw** [comment, marked as answer]
You found a critical bug. Fixed in latest.

**Reply by Dexteriority** [reply]
Thank you!
